dotnet/Microsoft.MacCatalyst.Sdk/targets/Microsoft.MacCatalyst.Sdk.props

```
@@ -2,6 +2,7 @@
 <Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <_PlatformName>MacCatalyst</_PlatformName>
+    <_PlatformRidNoArch>maccataylst</_PlatformRidNoArch>
```

I think this will fix the failing dotnettests (Mac Catalyst) test:
 
```suggestion
    <_PlatformRidNoArch>maccatalyst</_PlatformRidNoArch>
```